chart/compass/templates/tests/ord-aggregator/ord-aggregator-test.yaml

```diff
@@ -27,6 +27,7 @@ spec:
               - "oauth2.{{ .Values.global.ingress.domainName }}"
               - "compass-external-services-mock.compass-system.svc.cluster.local"
       {{ end }}
+      serviceAccountName: {{ $.Chart.Name }}-ord-aggregator
```

Comment:
The tests are in kyma-system namespace and this SA is in compass-system.

I am not sure, but it is possible that this will not work and you need to redefine the service account + role & rolebinding in the test chart in kyma-system as well.